pkg/rtc/participant.go

```diff
@@ -198,6 +202,7 @@ func NewParticipant(params ParticipantParams) (*ParticipantImpl, error) {
 			params.Telemetry),
 		supervisor:    supervisor.NewParticipantSupervisor(supervisor.ParticipantSupervisorParams{Logger: params.Logger}),
 		tracksQuality: make(map[livekit.TrackID]livekit.ConnectionQuality),
+		pacer:         pacer.NewPassThrough(params.Logger),
```

Comment:
`Pacer` is an interface to enqueue packets to be sent. Currently, I have two implementations
1. `pacer.PassThrough` -> functionally equivalent to how things are today. This one just immediately sends the packet in the calling go routine.
2. `pacer.NoQueue` -> this one sends all packets from another go routine. Use case here is to do TWCC (adding TWCC extension header and send timestamp the packet). Gives control over order of packet send, i. e. for bandwidth estimation/TWCC, would be good to ensure that send order matches the sequence number/timestamp order to give receive side a better chance of receiving in same order and get more stable bandwidth estimation.

NOTES:
1. There is more work needed like maybe completely bypassing audio tracks (or tracks which do not have TWCC extension) and send them immediately.
2. May need to implement more pacer types to meter data at channel capacity to prevent congestion.